chat.js

```diff
@@ -727,9 +729,15 @@ class CommandContext {
 				return false;
 			}
 
-			// If the corresponding config option is set, non-AC users cannot send links, except to staff.
-			if (Config.restrictLinks && !user.autoconfirmed && message.match(Chat.linkRegex)) {
-				if (!(targetUser && targetUser.can('lock'))) {
+			// If the corresponding Config option is set, non-AC users cannot send links, except to staff.
+			const links = message.match(Chat.linkRegex);
+			if (Config.restrictLinks && !user.autoconfirmed && links) {
+				let whitelisted = false;
+				for (let curLink of AC_LINK_WHITELIST) {
+					if (String(links).includes(curLink)) whitelisted = true;
```

Comment:
So `www.illegaldrugsandlewds.xxx/?pokemonshowdown.com` would be considered whitelisted by this?

Reply:
that is probably a problem